src/test/java/seedu/address/logic/commands/AddCommandTest.java

```diff
@@ -84,90 +81,6 @@ public void toStringMethod() {
         assertEquals(expected, addCommand.toString());
     }
 
-    /**
```

Comment:
Wait why was this deleted (87-170)

Reply:
I created a new class file for this because I need to use the Model stub for my property stuff as well